api/src/router/routes/profile.ts

```diff
@@ -42,8 +43,10 @@ router.delete('/:profileId/namespace/:namespaceId', asyncMiddleware(archiveProfi
 // Contacts
 router.get('/:profileId/contacts', asyncMiddleware(fetchProfileContacts));
 router.post('/:profileId/contact/:contactId', asyncMiddleware(addContactToProfile));
+router.post('/:profileId/contact-edit', asyncMiddleware(requestContactEdit));
 
 // Quota-edit
 router.get('/:profileId/quota-edit', asyncMiddleware(fetchProfileQuotaOptions));
 router.post('/:profileId/quota-edit', asyncMiddleware(requestProfileQuotaEdit));
+router.get('/:profileId/request', asyncMiddleware(fetchProfileEditRequests));
```

Comment:
Could this be on its own instead of being under the //quota-edit comment?

Reply:
Sure will, admittedly the API side was just placeholder to prove the WEB works. Will move this to an appropriate place.